### ISSUE TITLE
Port twisted.conch.openssh_compat.* to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8694]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8694|
|Type|task|
|Created|2016-07-27 07:05:37Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8694 8694
type__task task
reporter__scop scop
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1469603137182629 1469603137182629
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_2"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/427

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc commented|
|-|-|

In changeset 0445b48312e98bceed53b244c795ba19403fc0cb
```
#!CommitTicketReference repository="" revision="0445b48312e98bceed53b244c795ba19403fc0cb"
Add topfile for #8694
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 0d4655736e709b449708e3c7d90c6a371dffa35f
```
#!CommitTicketReference repository="" revision="0d4655736e709b449708e3c7d90c6a371dffa35f"
Merge pull request #428 from twisted/8694-scop-osshcompat-py3-1

Author: scop
Reviewer: rodrigc
Fixes: #8694


Port twisted.conch.openssh_compat.* to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
